modules/KalturaSupport/components/playlistAPI.js

```diff
@@ -325,7 +326,13 @@
 					}
 				}else{
 					var width = this.getConfig("fixedControls") ? $( ".playlistInterface" ).width() - this.getConfig("horizontalControlsWidth")*2 : $( ".playlistInterface" ).width();
-					this.setConfig( 'mediaItemWidth', Math.floor(width / this.getConfig("MinClips")) );
+					// decide width of the items
+					// If should fill remaining horizontal space use preconfigured 'MinClips'
+					// else use cached initial 'MinClips' config
+					var itemsNumber = this.getConfig('horizontalFillRemainingSpace') ?
+						this.getConfig('MinClips') : this.minClips;
```

Comment:
@amirch1 - this.getConfig('MinClips') stores current minClips config which may not be equal to the initial minClips passed to playlist plugin, on the other hand, this.minClips stores initial minClips value passed to the plugin (it's assigned in setup method here: https://github.com/vladm3/mwEmbed/blob/2.37-width-pl-param/modules/KalturaSupport/components/playlistAPI.js#L69)